easybuild/easyconfigs/r/RELION/RELION-2.0.1-intel-2016b.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '2.0.1'
+commit_id = 'a6225608e270'
```

Comment:
Ah, so the commit properly corresponds to version `2.0.1`, but there's no specific tarball for `2.0.1`?
Urgh...

How do we make sure the commit is bumped when the version is bumped?

Is there a simply RELION command we can use to verify the version? Then we can add a sanity check command for it?

Otherwise I'm 95% sure we'll forget to bump the commit and some point, and starting naming something 2.0.2 or 2.1.0 while it's actually still 2.0.1...